web/src/features/panels/ranking-panel/RankingPanel.tsx

```diff
@@ -52,8 +52,8 @@ export default function RankingPanel(): ReactElement {
   return (
     <div className="flex max-h-[calc(100vh-236px)] flex-col py-3 pl-4 pr-1 ">
       <div className="pb-5">
-        <div className="font-poppins text-lg font-medium">{t('ranking-panel.title')}</div>
-        <div className="text-sm">{t('ranking-panel.subtitle')}</div>
+        <h1>{t('ranking-panel.title')}</h1>
+        <h2 className="text-sm">{t('ranking-panel.subtitle')}</h2>
```

Comment:
This also impacts the size. But that might be fine?
Before:
![image](https://github.com/user-attachments/assets/73275d53-19f5-4e36-ae53-c2589dae072c)
Now:
![image](https://github.com/user-attachments/assets/ca320a80-b29b-4115-9633-d3f05a182f9a)

Reply:
@Alportan what do you think?

I think the H1 is fine but maybe the font should be changed in the h2, but it is a subtitle so maybe we should use the "title font"?

Reply:
So it's a custom, smaller H2? I think it works as it is. Don't have any strong preferences here 🙌

Reply:
Yeah it's a h2 but a tad smaller.

Mostly it's just so we have a h1, h2 and then the zone titles are h3 so we should have them all.

Reply:
Sounds good! All good on my side 👏